doc/developer-documentation.md

```diff
@@ -50,3 +50,72 @@ build_configs:
 
 That's it! The tree is enabled now.
 All the jobs defined under `jobs` section of [config file](https://github.com/kernelci/kernelci-pipeline/blob/main/config/pipeline.yaml) would run on the newly added tree until specified otherwise.
+
+
+## Enabling a new test
+
+KernelCI currently runs several test suites. This manual is intended
+to provide documentation for developers on how to enable new tests.
```

Comment:
nit: what is the line breaking convention for these docs - should it be per number of characters or dividing sentences (rest of the file suggests the latter)?

Reply:
The usual practice in markdown is to have around 80 characters per line. I have divided lines to follow that practice up to some extent.

Reply:
I believe context/sentence-based line breaking are a good choice for documents with change tracking because small adjustments or rewording result in an easy-to-follow diff in relevant lines only.

This being said - either way works for me.